chart/kubeapps/templates/ingress.yaml

```diff
@@ -1,38 +1,38 @@
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "kubeapps.fullname" . -}}
-{{- $ingressPath := .Values.ingress.path -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ $fullName }}
+  name: {{ template "kubeapps.fullname" $ }}
   labels:
-    app: {{ include "kubeapps.name" . }}
-    chart: {{ include "kubeapps.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-{{- with .Values.ingress.annotations }}
+    app: {{ template "kubeapps.name" $ }}
+    chart: {{ template "kubeapps.chart" $ }}
+    release: {{ $.Release.Name }}
+    heritage: {{ $.Release.Service }}
   annotations:
-{{ toYaml . | indent 4 }}
-{{- end }}
+    {{- if .Values.ingress.certManager }}
+    kubernetes.io/tls-acme: "true"
+    {{- end }}
+    {{- range $key, $value := .Values.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
```

Comment:
Why do you prefer this approach vs dumping everything like before? `{{ toYaml . | indent 4 }}`

Reply:
I just used because it's how we use it on others charts. Anyway, it's only adding the value of quoting the value...

I think it's irrelevant to use one or the another.

Reply:
> I just used because it's how we use it on others charts

ok, it makes sense. thanks for the explanation

We might need to move the rest of the chart to this approach in another PR since I believe that we use `{{ toYaml . | indent 4 }}` in other places.